init/bootchart.cpp

```diff
@@ -74,8 +74,12 @@ static void log_header() {
   std::string fingerprint = android::base::GetProperty("ro.build.fingerprint", "");
   if (fingerprint.empty()) return;
 
+#if !defined(ANDROID_CONTAINER)
   std::string kernel_cmdline;
   android::base::ReadFileToString("/proc/cmdline", &kernel_cmdline);
+#else
+  std::string kernel_cmdline(getenv("INIT_KERNEL_CMDLINE"));
+#endif
```

Comment:
AOSP master branch will clear environ vars, so the will then be an empty string. Since it's only for informative dumps, should be okay here.